### DISCUSSION TITLE
Bug

### DISCUSSION THREAD

**Discussion by LilSefGaming** [ScratchAddons/ScratchAddons#4439] [Help]
Pls fix the bug where notifications automatically Happen!

**Comment by WorldLanguages** [comment]
@smisamul6 Can you provide more details?

**Comment by LilSefGaming** [comment]
It shows the cpu notification late

**Reply by stretch07** [reply]
please elaborate

**Reply by Dhuls-Scratch** [reply]
What's CPU?

**Reply by CST1229** [reply]
They probably mean Windows/whatever OS they're using in this context

**Comment by LilSefGaming** [comment]
Computer

**Reply by BroJac5246** [reply]
If you want is to be able to help you, you need to give us enough detail. We need _at least_ a small paragraph explaining what the issue is and how to reproduce it. You could also just fill out an issue form to report a bug, which tells you what to fill in.

**Reply by stretch07** [reply]
They’re obviously trolling, or 5 years old
